app/jobs/import_twilio_data_job.rb

```diff
@@ -0,0 +1,25 @@
+class ImportTwilioDataJob < ActiveJob::Base
+  class InvalidDateParameter < StandardError
+    def initialize(msg, backtrace = nil)
+      set_backtrace(backtrace) if backtrace
+      super(msg)
+    end
+  end
+
+  queue_as :default
+
+  rescue_from(InvalidDateParameter) do |exception|
+    Bugsnag.notify(exception)
+  end
+
+  def perform(date_string)
+    date = parse_date(date_string)
+    DailyCalls::Twilio::Importer.new.import(start_date: date, end_date: date)
+  end
+
+  def parse_date(date_string)
+    Date.parse(date_string)
```

Comment:
I wonder if we couldn't simplify this by just rescuing from the `ArgumentError` that would be raised here? Sure, we wouldn't have the actual date string that caused the issue but I'm not sure that really helps us anyway. This scenario is pretty unlikely, so perhaps simplifying here makes sense?